modules/installation-special-config-mirrored-disk.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-mirrored-disk_{context}"]
+= Mirroring disks during installation
+
+During {product-title} installation, you can enable mirroring of the boot disk to two or more redundant replicas on master and worker nodes. A node stays active after failure as long as one replica still functions. Mirroring support is available on x86_64 nodes booted with BIOS or UEFI and on ppc64le nodes.
+
+Mirroring does not support replacement of a failed disk. To restore the mirror to a pristine, non-degraded state, reprovision the node. For disk encryption, an alternate method of configuring disk encryption is required.
```

Comment:
The last sentence is unrelated to the rest of the paragraph.  Let's just drop it; the instructions elsewhere should make this clear enough.